target/linux/x86/base-files/lib/preinit/43_grow_rootfspart

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+get_squashfs_size() {
+	local low high
+	low=$(hexdump -s 40 -n 4 -e '1/4 "%08x" "\n"' "$1")
+	high=$(hexdump -s 44 -n 4 -e '1/4 "%08x" "\n"' "$1")
+	echo $((0x$high$low))
+}
+
+run_grow_root_part() {
+    [ "$FAILSAFE" != "true" ] || return
+
+    local diskdev squashfs_magic
+    local SQUASHFS_ALIGN=$((0x10000))
+
+    . /lib/upgrade/common.sh
+
+    if export_bootdevice && export_partdevice diskdev 0; then
+	diskdev=/dev/$diskdev
```

Comment:
Your indent is somewhat arbitrary. Please check with some similar files and try to use it consistent to those (as far as possible). I'd just use tabs here ...

Reply:
I replaced all to tabs. It got messes while editing both with vim-full and busybox vim.